ios-xr/xrd-packer
# Move to AL2023

Update xrd-packer to use AL2023 base image.

Changes:
* Switch from yum to dnf
* Move TuneD forward to latest stable and remove Python2 dependencies
* Use `grubby` instead of TuneD bootloader plugin
* Update documentation

Testing:
* Verified host tuning in image using tuned verify, host-check and manual checking of hugepages/isolcpus (including on multi-numa)
* Ran VPET throughput and functionality suites (for vRouter)
